modules/ensembl/manifest/maker/main.nf

```diff
@@ -0,0 +1,57 @@
+// See the NOTICE file distributed with this work for additional information
+// regarding copyright ownership.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+process MANIFEST {
+    tag "$meta.accesion"
+    label 'adaptive'
+
+    conda "${moduleDir}/environment.yml"
+    container "ensemblorg/ensembl-genomio:v1.6.1"
+
+    input:
+        tuple val(meta), path(file_name)
+
+    output:
+        tuple val (meta), path("*", includeInputs: true)
+        path "versions.yml", emit: versions
+    
+    when:
+        task.ext.when == null || task.ext.when
+
+    script:
+        """
+        manifest_generate \
+            --manifest_dir .
+
+        schemas_json_validate \
+            --json_file manifest.json \
+            --json_schema "manifest"
```

Comment:
Im wondering how this will work without set up for access to the schema we have currently sat in [genomio data folder](https://github.com/Ensembl/ensembl-genomio/tree/main/src/python/ensembl/io/genomio/data/schemas)

Reply:
`usr/local/lib/python3.10/site-packages/ensembl/io/genomio/data/schemas/manifest.json` is the path location of this schema within the container image (Thanks Fatima you found this first)